source/GameInterface/Services/PartyBases/Patches/PartyBasePatch.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Common.Logging;
+using Common.Messaging;
+using GameInterface.Services.ItemRosters;
+using HarmonyLib;
+using Serilog;
+using TaleWorlds.CampaignSystem.Party;
+using TaleWorlds.CampaignSystem.Roster;
+
+namespace GameInterface.Services.PartyBases.Patches
+{
+    [HarmonyPatch(typeof(PartyBase))]
+    internal class PartyBasePatch
+    {
+        private readonly PartyBasePatch instance = new();
```

Comment:
Harmony patches do not require singletons, this is resolved using the harmony.PatchAll method

```suggestion
```